DigitalLearningSolutions.Data/Models/Progress.cs

```diff
@@ -15,5 +15,6 @@ public class Progress
         public int CustomisationVersion { get; set; }
         public int EnrollmentMethodId { get; set; }
         public int EnrolledByAdminId { get; set; }
+        public int? DiagnosticScore { get; set; }

```

Comment:
Why does this have to be nullable?

Reply:
The column in the database is nullable. I'm actually not sure what we should do with it if it's empty however.

Reply:
Good reason. And good question about the null case.
This is what the ticket says:

Reply:
@SteveJacksonSoft question related to this for 649 - do we want to display "N/A" if for the Progress.DiagnosticScore on the other progress pages, or is that specific to this page?

Reply:
Yep, checked with Kevin and he agrees, if we have null DiagnosticScore then display "N/A".